### ISSUE TITLE
WTForms 3.0.0 incompatability

### ISSUE THREAD

**Issue by rsyring** [level12/keg-elements#161]
```pytb
...
.tox/py39/lib/python3.9/site-packages/keg_elements/__init__.py:1: in <module>
    from . import forms  # noqa
.tox/py39/lib/python3.9/site-packages/keg_elements/forms/__init__.py:20: in <module>
    from wtforms_alchemy import (
.tox/py39/lib/python3.9/site-packages/wtforms_alchemy/__init__.py:21: in <module>
    from .fields import (  # noqa
.tox/py39/lib/python3.9/site-packages/wtforms_alchemy/fields.py:11: in <module>
    from wtforms.compat import string_types, text_type
E   ModuleNotFoundError: No module named 'wtforms.compat'
...

https://stackoverflow.com/questions/69879246/no-module-named-wtforms-compat

**Comment by rsyring** [comment]
Upstream issue: https://github.com/kvesteri/wtforms-alchemy/issues/158

**Comment by guruofgentoo** [comment]
Goalposts moved here - wtforms-alchemy has a new version out to resolve their incompatibility. Now we'll need to deal with our own breakage here in keg-elements: https://app.circleci.com/pipelines/github/level12/keg-elements/111/workflows/c4bd8f32-7552-45d5-a957-e7782379f514

**Comment by guruofgentoo** [comment]
Fixed in 0.7
